### PR TITLE
feat: add systemd service

### LINKED ISSUE #223: feature request: make daemon works as a systemd process
Can we make `acceld` as a systemd application ? We can load the config file in `/etc/acceld/config.yaml`. cc @imeoer.

### PR DESCRIPTION
**Pull request by Desiki-high**
1. add systemd service `acceleration-daemon.service`.
2. add make install for installing `acceleration-daemon.service`.

resolve #223.

### CODE REVIEW

File: misc/acceleration-daemon.service [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ExecStart=/usr/local/bin/acceld --config /etc/acceld/config.yaml
+Restart=always
+RestartSec=1
+KillMode=process
+OOMScoreAdjust=-999
+StandardOutput=journal
+StandardError=journal
+
+[Install]
+WantedBy=multi-user.target
```
Comment (imeoer) [inline misc/acceleration-daemon.service:18]: Add a blank line here.

File: misc/acceleration-daemon.service [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=acceleration-daemon
```
Comment (imeoer) [inline misc/acceleration-daemon.service:2]: How about `Acceleration Service Daemon`

File: Makefile [hunk 3]
```diff
@@ -2,6 +2,10 @@ VERSION_TAG=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
 GIT_COMMIT := $(shell git rev-list -1 HEAD)
 BUILD_TIME := $(shell date -u +%Y%m%d.%H%M)
 
+SOURCE_ACCELD_CONFIG ?= misc/config/config.nydus.yaml
```
Comment (imeoer) [inline Makefile:5]: `SOURCE_ACCELD_CONFIG` -> `ACCELD_CONFIG` ?

Comment (Desiki-high) [reply Makefile:5]: ALl fixed

**Review by imeoer — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**